sspl_test/plans/dev_sanity.pln

```diff
@@ -14,3 +14,4 @@ alerts.node.test_node_psu_actuator
 alerts.node.test_node_fan_actuator
 alerts.os.test_systemd_service_actuator
 alerts.node.test_node_bmc_interface
+alerts.os.test_sensor_recovery
```

Comment:
Please add this in alerts.pln also, as dev_sanity is subset of alert.pln file all tests present in dev_sanity should be present in alert.pln.

Reply:
Done.